pkg/cloud/openstack/cluster/actuator.go

```diff
@@ -27,6 +27,11 @@ func NewActuator(params providerv1openstack.ActuatorParams) (*Actuator, error) {
 }
 
 // Reconcile creates or applies updates to the cluster.
+// TODO: those are copied from original rbac_role.yaml, need remove them if not needed later
+// +kubebuilder:rbac:groups=openstackproviderconfig.k8s.io,resources=openstackmachineproviderconfigs,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Could you elaborate a bit more on how the generator works?

Does it parse *all* comments in the source tree looking for `kubebuilder` statements?

Reply:
@flaper87 the `kubebuilder` documentation for the generator can be found here: https://book.kubebuilder.io/beyond_basics/generating_crd.html